dm/worker/server.go

```diff
@@ -460,15 +422,20 @@ func (s *Server) QueryWorkerConfig(ctx context.Context, req *pb.QueryWorkerConfi
 func (s *Server) MigrateRelay(ctx context.Context, req *pb.MigrateRelayRequest) (*pb.CommonWorkerResponse, error) {
 	log.Infof("[server] receive MigrateRelay request %+v", req)
 
+	err := s.worker.MigrateRelay(ctx, req.BinlogName, req.BinlogPos)
+	if err != nil {
+		log.Errorf("[worker] %v MigrateRelay error %v", req, errors.ErrorStack(err))
+	}
+	return mkResponse(err), nil
+}
+
+func mkResponse(reqErr error) *pb.CommonWorkerResponse {
```

Comment:
is `mkCommonWorkerResponse` more accurate ?

Reply:
Please spell out "make" instead of "mk".